services/parser-database/connector.py

```diff
@@ -73,8 +101,10 @@ def save_keywords_in_memory(keywords, article):
         if keyword not in keywords_in_memory:
             keywords_in_memory[keyword] = []
         keywords_in_memory[keyword].append({
+            "id": article["id"],
             "articleNumber": article["articleNumber"],
```

Comment:
nit: Update all references of `articleNumber` to `number` since that was the one of the agreed attributes to interface with the database.

Otherwise, this'd break the backend and frontend services.

The complete response format the backend is expecting should look like this:
```
Article {
 id: ID <- A String
 number: Int
 content: String
 keywords: [String]
 wordCount: Int
}
```

Reply:
Please don't break my code.

Reply:
Resolved as per oscar's private request

Reply:
I need this @jaimehisao HAHAHAH.

Gimme my attributes!!!!!